integration-test/data/FoD/fod-integation-test-data.json

```diff
@@ -10,8 +10,8 @@
 	"expectOutput": "default"
   },  
   {
-    "commands": "fod application create-web-app",

```

Comment:
Why version of fcli are you targetting? `fod application create-web-app` is correct for 2.0.0 onwards - which is what we should be targetting. It looks like you have reverted to 1.2.5 with `--app-type=Web`?